### ISSUE TITLE
Disable Image Proxy request completely

### ISSUE THREAD

**Issue by vilbergs** [froala/wysiwyg-editor#4562]
Hello! 

We have a custom upload solution for our Froala Editor implementation and we're running into problems with the CORS proxy request that gets made when images are pasted into the editor.

In cases where the request to the proxy server fails, the `beforePasteUpload` event isn't called and the image gets pasted into the editor with its original url. This is bad as we only want image urls from our own servers to be in the editor content, and we rely on the beforePasteUpload event for that.

##### Expected behavior.

Image proxy requests can be disabled.

##### Actual behavior.

When pasting an image into the editor, a request is always made to https://cors-anywhere.froala.com

##### Steps to reproduce the problem.
1. copy an image
2. paste the image into the editor
3. The request to the proxy server shows up in the network de-tools tab.

##### Editor version.
4.0.16

##### OS.
MacOS.

##### Browser.
Chromium browsers have been tested

##### Recording.
Happy to add a recording if the problem is hard to follow.

**Comment by ilyaskarim** [comment]
Please configure the following API options to disable/change the Proxy URL: https://froala.com/wysiwyg-editor/docs/options#imageCORSProxy
